cleanlab/internal/label_quality_utils.py

```diff
@@ -85,19 +85,16 @@ def get_normalized_entropy(pred_probs: np.ndarray, min_allowed_prob: float = 1e-
     Parameters
     ----------
     pred_probs : np.array (shape (N, K))
-      P(label=k|x) is a matrix with K model-predicted probabilities.
       Each row of this matrix corresponds to an example x and contains the model-predicted
-      probabilities that x belongs to each possible class.
-      The columns must be ordered such that these probabilities correspond to class 0,1,2,...
-      `pred_probs` should have been computed using 3 (or higher) fold cross-validation.
+      probabilities that x belongs to each possible class: P(label=k|x)
 
     min_allowed_prob : float, default=1e-6
       Minimum allowed probability value. Entries of `pred_probs` below this value will be clipped to this value.
       Ensures entropy remains well-behaved even when `pred_probs` contains zeros.
     Returns
     -------
     entropy : np.array (float)
-
+      Each element is the normalized entropy of the corresponding row of ``pred_probs``.
     """
 
     num_classes = pred_probs.shape[1]
```

Comment:
This should also call `get_num_classes()`

Reply:
I actually purposefully left the num_classes determination here based on pred-probs because here the "number of classes" is not the same as the official number of classes the cleanlab package determines in a user's dataset (note this function doesn't even take in any user data/labels).  In this function, the `num_classes` variable is just a placeholder the number of columns of `pred_probs.shape`